python/src/aac/lang/language_context.py

```diff
@@ -83,15 +95,40 @@ def add_definitions_to_context(self, definitions: list[Definition]):
         Args:
             definitions: The list of Definitions to add to the context.
         """
-        extension_definitions = get_definitions_by_root_key("ext", definitions)
+
+        def dirty_dependency_sort_for_definitions_with_inheritance(definitions: list[Definition]) -> list[Definition]:
```

Comment:
What does "dirty" mean, here? Is it a reference to the possibility of duplicated parent definitions? If so, could we eliminate that using a set?

Reply:
Dirty as in simple-fast. This is a superficial implementation until we can actually calculate dependency order for definitions. I can rename dirty -> simple here.

Reply:
Understood. It's fine to leave it as - I just wasn't sure in what way this was "dirty".